xbmc/pvr/PVRManager.cpp

```diff
@@ -97,6 +97,7 @@ CPVRManager::CPVRManager(void) :
     m_timers(NULL),
     m_addons(NULL),
     m_guiInfo(NULL),
+    m_parentalTimer(nullptr),
```

Comment:
sure, I need to touch it again anyway. PVRManager destroys addons that are in use. i.e. when having open a gui window owned by an addon and closing the app.